frontend/express/public/core/home/stylesheets/_main.scss

```diff
@@ -1,3 +1,9 @@
 #main_home_view .el-loading-parent--relative {
 	min-height: 100px;
+}
+#main_home_view .bu-level-left:first-child > h2 {
+	font-size: 20px !important;
+}
+.cly-cmp-28 {
```

Comment:
"cly-cmp-" is not a real, permanent class, it is just a temporary one which is added by App runtime. So we cannot use it for addressing an element. If there is no class assigned to the element you're trying to style, you should assign a class to the element yourself :slightly_smiling_face: